.gitignore

```diff
@@ -0,0 +1,2 @@
+yarn.lock
```

Comment:
yarn.lock must be in the repo

Reply:
Solved :raised_hands:

Reply:
> @fual hey, what about merge?)

thank you! That's a best pull request!
But please write the tests and resolve conficlts, please